src/main/java/mods/railcraft/common/carts/EntityTunnelBore.java

```diff
@@ -857,6 +865,7 @@ protected float getLayerHardness(BlockPos targetPos, BlockRailBase.EnumRailDirec
             IBoreHead head = (IBoreHead) boreSlot.getItem();
             float dig = 2f - head.getDigModifier();
             hardness *= dig;
+            hardness /= (1 + EnchantmentHelper.getEnchantmentLevel(Enchantments.EFFICIENCY, boreSlot));
```

Comment:
"If a proper tool is used, the tool speed multiplier is further increased by Efficiency. If the level of Efficiency is not 0, then the level squared plus 1 is added to the tool speed. For example, Efficiency I adds 2 to the value, while Efficiency V adds 26."

That doesn't sound much like what you implemented.